A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -830,6 +848,34 @@ var A320_Neo_UpperECAM;
             this.divMain.appendChild(this.gaugeN1);
             this.allGauges.push(this.gaugeN1);
         }
+        getModeEGTMax(){
+            if(this.throttleMode == ThrottleMode.TOGA && this.timerTOGA > 0){
+                return 1060;
+            }
+            else if(this.timerTOGA <0){
+                return 1025;
+            }
```

Comment:
how would the execution ever get past this when not in TOGA?
if its not in TOGA, then `this.timerTOGA` is going to be -1, so it will always return 1025.

Reply:
The region is for it to be in TOGA, if you exceed TOGA timer it falls back to MCT limit i.e 1025 instead of 1060.